bundles/org.openhab.voice.pipertts/src/main/java/org/openhab/voice/pipertts/internal/PiperTTSService.java

```diff
@@ -93,17 +113,89 @@ public PiperTTSService(final @Reference TTSCache ttsCache) {
 
     @Activate
     protected void activate(Map<String, Object> config) {
-        try {
-            piper = new PiperJNI();
-            piper.initialize(true, false);
-            logger.debug("Using Piper version {}", piper.getPiperVersion());
-        } catch (IOException e) {
-            logger.warn("Piper registration failed, the add-on will not work: {}", e.getMessage());
-        }
+        activateTask = executor.submit(() -> {
+            try {
+                setupNativeDependencies();
+                piper = new PiperJNI();
+                piper.initialize(true, false);
+                logger.debug("Using Piper version {}", piper.getPiperVersion());
+                ready = true;
+            } catch (IOException e) {
+                logger.warn("Piper registration failed, the add-on will not work: {}", e.getMessage());
+            }
+        });
         tryCreatePiperDirectory();
```

Comment:
You must move this line before starting your thread activateTask.You need first the PIPER_FOLDER to be able to create the LIB_FOLDER.